third_party/terraform/resources/resource_google_folder_organization_policy.go

```diff
@@ -27,6 +32,17 @@ func resourceGoogleFolderOrganizationPolicy() *schema.Resource {
 	}
 }
 
+func resourceFolderOrgPolicyImporter(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
```

Comment:
Should we use a new-style (the MM kind) of importer instead? We've had users expect similar import support in handwritten resources and ask for us to backport newer importers.

Reply:
Do you mean using `ParseImportID([]string{"projects/(?P<project>[^/]+)/...` to support multiple types of import formatting?

Reply:
Yep!